src/preview.ts

```diff
@@ -20,6 +27,8 @@ export class PreviewPlugin implements Plugin {
         variantName: 'variant',
     };
 
+    public static readonly NONE_AUDIENCE = 'None';
```

Comment:
I guess internally we can call "NO_AUDIENCE", I'm not very sure though

```suggestion
    public static readonly NO_AUDIENCE = 'None';
```

Reply:
There is a difference between "no audience" and the "none audience". "No audience" is when no audience is informed. "None audience" is when I inform I want to see the content out of any audience. Is there a slight but relevant difference. Did you get it?

Reply:
Yes, thank you!